LandSandBoat/server
# Remove deprecated BCNM loot handling

**_I affirm:_**

## What does this pull request do?
Now that all BCNMs that have armoury crates have been converted to Interaction Framework, remove old sql, core and lua functions that are no longer necessary



## Steps to test these changes
Check BCNMs that have loot (both addEssential and armouryCrate defined variants) to ensure battlefields are still rewarding as expected.
